cime_config/machines/config_machines.xml

```diff
@@ -1426,6 +1426,84 @@
     </environment_variables>
   </machine>
 
+
+  <machine MACH="boca">
+    <DESC>SNL clust</DESC>
+    <NODENAME_REGEX>(boca)</NODENAME_REGEX>
+    <OS>LINUX</OS>
+    <PROXY>proxy.sandia.gov:80</PROXY>
+    <COMPILERS>intel</COMPILERS>
+    <MPILIBS>openmpi</MPILIBS>
+    <PROJECT>fy220022</PROJECT>
+    <SAVE_TIMING_DIR>/projects/ccsm/timings</SAVE_TIMING_DIR>
+    <SAVE_TIMING_DIR_PROJECTS>.*</SAVE_TIMING_DIR_PROJECTS>
+    <CIME_OUTPUT_ROOT>/gpfs/$USER/acme_scratch/boca</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/projects/ccsm/inputdata</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>/projects/ccsm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
+    <DOUT_S_ROOT>$CIME_OUTPUT_ROOT/archive/$CASE</DOUT_S_ROOT>
+    <BASELINE_ROOT>/projects/ccsm/ccsm_baselines/$COMPILER</BASELINE_ROOT>
+    <CCSM_CPRNC>/projects/ccsm/cprnc/build.toss3/cprnc</CCSM_CPRNC>
+    <GMAKE_J>8</GMAKE_J>
```

Comment:
Only 8? I have never logged in on boca, so I don't know what I'm talking about, but doesn't boca have more cores/node?

Reply:
if this builds on the login node, probably don't want to use 36. 12 or 18 might be okay. Looks like there's two login nodes.

Reply:
Good catch, I will increase this.

Reply:
Actually, there's 36 per node, but I think builds would still happen on a shared login node. Might not be responsible usage to max out the parallel build with other users on the login nodes. Thoughts?

Reply:
Depending on CPU type, builds can be quite fast. I don't think it's bad to use 90% of cores if you only keep most of them busy for a minute or so. Especially if we use make (rather than ninja): make proceeds by targets (rather than objects), so if you run make -jN, you would use ~N processes for t<<100% of the time.

I would use 32.

Reply:
sounds good. It could always be changed if someone complains.

Reply:
Alright, done.